config/FindTAO.cmake

```diff
@@ -193,7 +193,7 @@ MACRO(ORO_ADD_CORBA_SERVERS _sources _headers)
       GET_FILENAME_COMPONENT(_filedir ${_tmp_FILE} PATH)
 
       SET(_server  ${CMAKE_CURRENT_BINARY_DIR}/${_basename}S.cpp)
-      SET(_serverh ${CMAKE_CURRENT_BINARY_DIR}/${_basename}S.h ${CMAKE_CURRENT_BINARY_DIR}/${_basename}S.inl)
+      SET(_serverh ${CMAKE_CURRENT_BINARY_DIR}/${_basename}S.h)
```

Comment:
This change was required on my machine with TAO 2.3.1 (Ubuntu 14.04 x64 with default compilers) to fix cmake errors when it tries to install non-existent files. I am not sure if older versions of TAO generate the server inline skeleton files and if they really need to be installed. At the end, this is the only effect of adding them to the `_serverh` variable here.